include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -288,9 +289,27 @@ namespace grl { namespace robot { namespace arm {
 
           auto name = fbbP->CreateString(std::get<RobotName>(params_));
 
-          auto kukaiiwaArmConfiguration = flatbuffer::CreateKUKAiiwaArmConfiguration(*fbbP,name,commandInterface_,monitorInterface_);
+          auto clientCommandMode = grl::flatbuffer::EClientCommandMode_POSITION;
```

Comment:
yes EClientCommandMode is not used anywhere in GRL_Driver.java. Unless there is another driver which vrep uses I think this is safe.